docs/using-hive.md

```diff
@@ -188,19 +188,47 @@ oc get nodes
 
 Hive can optionally create delegated Route53 DNS zones for each cluster.
 
-To use this functionality you must first define the base domains you want to manage DNS for in your HiveConfig.
+To use this feature:
```

Comment:
Should we create a sub topic for AWS as this very specific to it. We are going to support other cloud providers and the text will be different for them.

Reply:
As soon as we have another provider to document yes. Probably a separate markdown file at that point.

Reply:
Added a note.